src/main/java/step3/domain/LottoNumber.java

```diff
@@ -1,29 +1,38 @@
 package step3.domain;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Objects;
 
 public class LottoNumber implements Comparable<LottoNumber> {
 	private final int number;
 
-	public static final int MIN_LOTTO_VALUE = 1;
-	public static final int MAX_LOTTO_VALUE = 45;
-
+	private static final int MIN_LOTTO_VALUE = 1;
+	private static final int MAX_LOTTO_VALUE = 45;
+	private static final List<LottoNumber> LOTTO_VALUES = new ArrayList<>();
```

Comment:
```suggestion
	private static final List<LottoNumber> LOTTO_VALUES = IntStream.rangeClosed(MIN_LOTTO_VALUE, MAX_LOTTO_VALUE)
			.boxed()
			.map(LottoNumber::new)
			.collect(Collectors.toList());
```
스트림을 활용한다면 이렇게도 할수 있겠네요 !